app/models/time_profile.rb

```diff
@@ -149,22 +150,6 @@ def match_user_tz?(user_id, user_tz)
         (profile_type == "user" && profile_key == user_id))
   end
 
-  private
```

Comment:
The class methods below are expected to be private (I understand they technically aren't, and we shouldn't change them in this PR, but moving this line changes the expectation because they now appear to be intentionally public).

Reply:
It wasn't clear to me, especially as `self.default_time_profile` has some documentation, while the others don't.

Reply:
Adding more doubt, marking those methods as private breaks the specs as written. Except for 'default_time_profile`, making any of them private breaks the specs.

Reply:
I think marking those as private and using send in the specs may have merit.
using a helper function for the tests to make the send and the intent more clear may be reasonable
(sorry jason - I know you tend to not like this)

I feel like I've pulled the private test trick - sorry if those were mine.

Reply:
If the consensus is for us to keep them private and use `send` in the specs, I'm ok with that. Just let me know. :)

Reply:
Yes, I think that's a good approach, but not for this PR.  IMO, a PR that does "rubocop fixes" should not change any semantics (even if that's needed to fix rubocop).  Instead there should be one clean PR with most of the cops fixed but no semantics issues changed, then a separate PR that fixes semantic issues.

I would leave this particular change out of this PR and do it in a follow up PR where the semantic change can be tested properly.

Reply:
Ok, undid it for this PR.